src/Adapters/Auth/gcenter.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+// Helper functions for accessing the google API.
+//var Parse = require('parse/node').Parse;
+var crypto = require('crypto');
+var request = require('request');
+var url = require('url');
+
+// Returns a promise that fulfills if this user id is valid.
+function validateAuthData(authData)
+{
```

Comment:
bring this opening brace back up to the function declaration, keeping in style with the rest of everything.